### ISSUE TITLE
[New Feature]: (hopefully quite easy to implement) Adding a quick way to simulate NoScript default set/change of an automatically identified web server or 'domain' trust script level

### ISSUE THREAD

**Issue by SwimmeRM** [hackademix/noscript#357]
After doing a quick search on forum side I do hope that implementing this that I consider a New Feature might be quite quick and easy to do. So, it would consist in adding a quick way to simulate NoScript own default initial identification of a script web server or 'domain' that is made to then let user decide which trust level to set/change as most appropriate for an automatically identified script web server/'domain'.
This need came out while completing yesterday (Monday), some additional troubleshooting on a quite weird effect found the day before (Sunday), when I verified that for some web sites a script identified 'domain' may only appear in NoScript scripts sites identified list only the 1st time own web browser will open a certain and well-known known web page, but will no more appear/be identified during any additional visit (or page refresh) to same page.
- 1st screenshot I'm adding here as an example is related to a very well-known site and I took it from a Microsoft Edge InPrivate session (=equivalent to an Incognito mode one from Google Chrome/Chromium) during 1st visit to involved page and so this issue might also possibly be related to the quite common fact that most cookies are set during 1st visit to a web site or page. Anyway, this example is indeed good also for considering related effects found even while more simply troubleshooting access to 1 or more specific web site pages (so not only this well-known one I was involved with);
![Amazon_it_Elenco_siti_che_attivano_Script_mostrato_da_NoScript_nella_sessione_InPrivate_di_MS_Edge_(=Incognito_di_Google)_#14](https://github.com/hackademix/noscript/assets/102351974/da9e2538-f1e1-4fd9-b3b4-0bb46ba27342)

- 2nd screenshot is instead taken from another but this time more 'standard' Microsoft Edge session when accessing same page for at least the 2nd time and after that during 1st visit some a more appropriate script trust level choice was not done for a particularly identified script web server or 'domain'.
![Amazon_it_Elenco_siti_che_attivano_Script_mostrato_da_NoScript_in_una_sessione_standard_di_MS_Edge_ma_dopo_non_aver_fatto_alcuna_scelta_durante la_1^_visita_ad_1_pagina#15](https://github.com/hackademix/noscript/assets/102351974/882757e3-4ec2-4265-b657-42d9279dca1c)

Basically, because for '...amazonws.com' a better script trust level was not choosen during 1st visit, and same site was no more identified by NoScript during successive visits, I've also tried to add it manually using NoScript Options but there I could only insert 'https://amazonaws.com' and so its resulting entry was different from '...amazonaws.com' that after that I would have liked to see listed, just like if I correctly choose a most appropriate trust script level for it during my 1st visit.

**P.S.** Just in case knowing might be needed, used OS was Windows 7 SP1 x64 Ultimate.
